src/content/get-started/transfer-a-sparsified-model.mdx

```diff
@@ -15,7 +15,7 @@ SparseZoo and SparseML work together to accomplish this goal:
 - SparseML contains convenient training CLIs that run transfer-learn while preserving the same level of sparsity as the starting model
 
 By fine-tuning pre-sparsified models onto your dataset, you can avoid the time, money, and hyperparameter tuning involved with sparsifying a dense model from scratch.
-Once trained, deploy your model on the DeepSparse Engine for GPU-level performance on CPUs.
+Once trained, deploy your model with DeepSparse for GPU-level performance on CPUs.
```

Comment:
Since DeepSparse describes the suite of components, are you saying one can deploy with the whole suite? Are all four components doing the deployment?